bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/internal/ThingManager.java

```diff
@@ -140,23 +144,19 @@ public void statusUpdated(Thing thing, ThingStatusInfo thingStatus) {
             if(thing.getStatusInfo().equals(thingStatus)) {
                 return;
             }
-            
-            thing.setStatusInfo(thingStatus);
+            setThingStatus(thing, thingStatus);
             logger.debug("Status of {} changed to {}", thing.getUID(), thingStatus.toString());
```

Comment:
This log message is redundant due to the EventLogger and should thus be removed.